### ISSUE TITLE
Optimizing metadata loading (detecting unique identifiers etc...) (mariadb)

### ISSUE THREAD

**Issue by itschrono** [dbeaver/dbeaver#13863]
**Is your feature request related to a problem? Please describe.**
Way too slow query involving  INFORMATION_SCHEMA.CHECK_CONSTRAINTS and INFORMATION_SCHEMA.TABLE_CONSTRAINTS.

**Describe the solution you'd like**
The current query is:
```sql
SELECT cc.CONSTRAINT_NAME, cc.CHECK_CLAUSE, tc.TABLE_NAME
FROM INFORMATION_SCHEMA.CHECK_CONSTRAINTS cc, INFORMATION_SCHEMA.TABLE_CONSTRAINTS tc
WHERE cc.CONSTRAINT_NAME = tc.CONSTRAINT_NAME
AND cc.CONSTRAINT_SCHEMA = 'schema' AND tc.TABLE_NAME = 'table'
ORDER BY cc.CONSTRAINT_NAME;
```
it takes forever when having a lot of databases and tables (because I think it is reading them all on disk).
The way we could do it (I think it is equivalent):
```sql
SELECT cc.CONSTRAINT_NAME, cc.CHECK_CLAUSE, tc.TABLE_NAME
FROM (SELECT CONSTRAINT_NAME, CHECK_CLAUSE
FROM INFORMATION_SCHEMA.CHECK_CONSTRAINTS
WHERE CONSTRAINT_SCHEMA = 'schema'
ORDER BY CONSTRAINT_NAME) cc,
(SELECT TABLE_NAME, CONSTRAINT_NAME
FROM  INFORMATION_SCHEMA.TABLE_CONSTRAINTS
WHERE TABLE_SCHEMA = 'schema' AND  TABLE_NAME = 'table') tc
WHERE cc.CONSTRAINT_NAME = tc.CONSTRAINT_NAME
ORDER BY cc.CONSTRAINT_NAME;
```

**Describe alternatives you've considered**
N/A

**Additional context**
![image](https://user-images.githubusercontent.com/25153662/132832457-a8e49114-565b-4578-9025-f97becfb7d4a.png)

![image](https://user-images.githubusercontent.com/25153662/132832493-41e2438f-1ce7-424f-8f95-71107c58b372.png)

**Comment by Matvey16** [comment]
@itschrono Thank you for the suggestion

**Comment by LonwoLonwo** [comment]
Query changed. Implemented in DBeaver 21.2.5

**Comment by Matvey16** [comment]
Verified
